modules/machineset-non-guaranteed-instance.adoc

```diff
@@ -3,30 +3,21 @@
 // * machine_management/creating_machinesets/creating-machineset-aws.adoc
 
 [id="machineset-non-guaranteed-instance_{context}"]
-= MachineSet Spot Instances
+= MachineSets that deploy machines as Spot Instances
 
-You can save on costs by creating an AWS MachineSet that deploys machines as non-guaranteed Spot Instances.
-Spot Instances utilize unused AWS EC2 capacity and are less expensive than On-Demand Instances.
-You can use Spot Instances for workloads that can tolerate interruptions, such as batch or stateless,
-horizontally scalable workloads.
+You can save on costs by creating a MachineSet running on AWS that deploys machines as non-guaranteed Spot Instances. Spot Instances utilize unused AWS EC2 capacity and are less expensive than On-Demand Instances. You can use Spot Instances for workloads that can tolerate interruptions, such as batch or stateless, horizontally scalable workloads.
 
 [IMPORTANT]
 ====
-It is strongly recommended that control plane machines are not created on Spot Instances
-due to the increased likelihood of the instance being terminated. Manual intervention is
-required to replace a terminated control plane node.
+It is strongly recommended that control plane machines are not created on Spot Instances due to the increased likelihood of the instance being terminated. Manual intervention is required to replace a terminated control plane node.
 ====
 
-AWS EC2 can terminate a Spot Instance at any time. AWS gives a two-minute warning to
-the user when an interruption occurs. The {product-title} begins to remove the workloads
-from the affected instances when AWS issues the termination warning.
+AWS EC2 can terminate a Spot Instance at any time. AWS gives a two-minute warning to the user when an interruption occurs. {product-title} begins to remove the workloads from the affected instances when AWS issues the termination warning.
 
-Interruptions can occur when using Spot Instances if:
+Interruptions can occur when using Spot Instances for the following reasons:
 
-* The instance price exceeds your maximum price.
-* The demand for Spot Instances increases.
-* The supply of Spot Instances decreases.
+* The instance price exceeds your maximum price
+* The demand for Spot Instances increases
+* The supply of Spot Instances decreases
 
-When AWS terminates an instance, a termination handler running on the Spot Instance
-node deletes the machine resource. To satisfy the MachineSet `replicas` quantity, the
-MachineSet creates a new machine that then requests a new Spot Instance.
+When AWS terminates an instance, a termination handler running on the Spot Instance node deletes the machine resource. To satisfy the MachineSet `replicas` quantity, the MachineSet creates a machine that requests a Spot Instance.
```

Comment:
Related to my previous comment, I think it should be "the MachineSet _object_ creates" or just "the machine set creates".